app/schemas/interactions/nudged.avsc

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "interactions.nudged",
+  "type": "record",
+  "fields": [
+    {
+      "name": "uuid",
+      "type": "string"
+    },
+    {
+      "name": "app",
+      "type": "string"
+    },
+    {
+      "name": "target",
+      "type": "string"
+    },
+    {
+      "name": "context",
+      "type": "string"
+    },
+    {
+      "name": "flavor",
+      "type": "string"
+    },
+    {
+      "name": "medium",
+      "type": "string"
+    },
+    {
+      "name": "at_time",
+      "type": "string"
```

Comment:
can we get a maximally compact timestamp representation to use all over the place?  Like milliseconds since epoch, https://temporenc.org/, etc...

Reply:
also `at_time` may be redundant, but probably good to record what this timestamp actually means, like is it recording when the event happened in the browser?  See https://openstax.slack.com/archives/C01498TCQ77/p1598989665039300 too

Reply:
@reedstrm do we need an explicit timestamp in here like this?

Reply:
Let's assume we're usually going to have a client-provided timestamp, explicitly in the payload schema. Most use cases need at least this.

 However, again, the specification of a particularly involved timestamp format is perhaps overkill. This is starting to feel like XML-SOAP or even RPC, rather than HTTP/REST. (Not in details, but in complexity of specification, and requiring opaque client libraries to interact with)